.github/workflows/linux.yml

```diff
@@ -98,36 +98,36 @@ jobs:
     secrets: inherit
     with:
       unique-id: ${{ needs.runner-select.outputs.unique-id }}
-      is-self-hosted: ${{ fromJSON(needs.runner-select.outputs.is-self-hosted) }}
+      is-self-hosted: ${{ runner.environment == 'self-hosted' }}
```

Comment:
Is this change correct? Won't the runner-timeout job itself be ran on github hosted runners?

Reply:
Oh, thanks for catching that! I was running search and replace and apparently didn't pay enough attention to the context :(